README.md

```diff
@@ -104,18 +95,61 @@ The lighthouse report results are automatically printed to the **Deploy log** in
 2:35:17 PM: }
 ```
 
-We currently support the following settings, which are passed directly to Lighthouse:
+## Lighthouse plugin configuration options
+
+### Run Lighthouse for different site paths
+
+To understand how your site's performance compares across different parts of your site, you can specify different site paths where you'd like to run Lighthouse audits.  Updates to `netlify.toml` will take effect for new builds.
+
+### Example to audit a site's root path and the contact page
+
+```
+[[plugins]]
+  package = "@netlify/plugin-lighthouse"
+
+  # Generate a Lighthouse report for the site's root path
+  [[plugins.inputs.audits]]
+  path = ""
+
+  # Generate a Lighthouse report for the contact site path
+  [[plugins.inputs.audits]]
+  path = "contact"
+
+```
```

Comment:
This could/is misleading. Actually `path` is the path for the directory that will be served by the express app and served in the browser URL. That means the users will be only able to test the contact page if the folder structure is `contact/index.html`. And, if not wrong, all the assets (css, images, js..) outsite this folder won't be reachable. IMO we should remove this section all together until we implement a solution to generate reports for different URLS

Reply:
cc @MetalMatt-Netlify on this.

Reply:
> That means the users will be only able to test the contact page if the folder structure is contact/index.html

I could add that this folder structure is part of this example or we can remove this. I wonder if you think we should remove the entire section or just the example without the relevant folder structure for the site.

Reply:
It is misleading talk about 'contact page', we are already seeing people (and ourselves) being confused by this `path` property and expecting a different behaviour. 
We can keep it replacing the concept of contact page by contact directory.

Reply:
> It is misleading talk about 'contact page', we are already seeing people (and ourselves) being confused by this `path` property and expecting a different behaviour. We can keep it replacing the concept of contact page by contact directory.

Oh yes, good call. This should only reference how to configure multiple audits for directories, but not for specific files / pages (yet)

Reply:
Per discussion today, sounds like we will remove this section for now.